tests/comparisons/Fixture/testImportRecordsFromDatabase.php

```diff
@@ -52,4 +54,6 @@ class UsersFixture extends TestFixture
             'updated' => '2012-06-12 01:24:31'
         ],
     ];
```

Comment:
The failed test is due to inadequate indentation of the records here. If anyone has a hint on the proper/best way to address this, I'm all ears.

Reply:
You need one level more, indeed.
Should be doable on the part where it generates it.